config/locales/en.yml

```diff
@@ -81,7 +81,7 @@ en:
 
   project_module_meetings: "Meetings"
 
-  text_in_hours: "in hours"
```

Comment:
You delete this, but is it not still used just a couple of lines [below](https://github.com/finnlabs/openproject-meeting/blob/dev/app/views/meetings/_form.html.erb#L84)?